spring-cloud-circuitbreaker-resilience4j/src/main/java/org/springframework/cloud/circuitbreaker/resilience4j/Resilience4JCircuitBreakerFactory.java

```diff
@@ -126,10 +130,17 @@ private Resilience4JCircuitBreaker create(String id, String groupName,
 			ExecutorService circuitBreakerExecutorService) {
 		Resilience4JConfigBuilder.Resilience4JCircuitBreakerConfiguration config = getConfigurations()
 				.computeIfAbsent(id, defaultConfiguration);
-		return new Resilience4JCircuitBreaker(id, groupName, config.getCircuitBreakerConfig(),
-				config.getTimeLimiterConfig(), circuitBreakerRegistry, timeLimiterRegistry,
-				circuitBreakerExecutorService, Optional.ofNullable(circuitBreakerCustomizers.get(id)),
-				bulkheadProvider);
+		if (resilience4JConfigurationProperties.isDisableThreadPool()) {
+			return new Resilience4JCircuitBreaker(id, groupName, config.getCircuitBreakerConfig(),
```

Comment:
I would add another constructor that doesn't take an executor service

Reply:
Created a new constructor.